src/Wallabag/CoreBundle/Repository/EntryRepository.php

```diff
@@ -13,77 +13,66 @@ class EntryRepository extends EntityRepository
      * Retrieves unread entries for a user.
      *
      * @param int $userId
-     * @param int $firstResult
-     * @param int $maxResults
      *
      * @return Paginator
```

Comment:
Could you adjust the docblock for every function? It now returns a `DoctrineORMAdapter`.
Also, you can remove the `use Doctrine\ORM\Tools\Pagination\Paginator;`